app/controllers/v1/doctors_controller.rb

```diff
@@ -17,12 +18,16 @@ def create
   def destroy
     @doctor = Doctor.find(params[:id])
     if @doctor.destroy
-      render json: { status: 201, message: 'doctor has deleted successfully!', content: { doctor: @doctor } }
+      render json: { status: 201, message: 'doctor has been deleted successfully!', content: { doctor: @doctor } }
```

Comment:
Hi @kosher9, 
line 20 and line 21 is the same please fix it

Reply:
Hey @samiullahbahadur, they are not the same, line 20 has been changed. That's why it is red.

Reply:
Check it well please, they look like they are the same but they are quite different.